05/GeoffRiley/similar_tweeters.py

```diff
@@ -0,0 +1,63 @@
+"""
+PyBites challenge 05: https://codechalleng.es/challenges/5/
+"""
+import re
+import string
+import sys
+
+from gensim import corpora, models, similarities
+import nltk
+from nltk.corpus import stopwords
+from user_tweets import UserTweets
+
+nltk.download('stopwords')
+sw = set(stopwords.words('english'))
+
+# exclusions:
+#  1. Tags starting with '#'
+#  2. email addresses containing '@'
+#  3. web addresses containing 'http' or 'https'
+#  4. URL indicators beginning '//'
+#  5. numbers comprising one or more digits followed by an optional two alphas
+exclusions = re.compile(r'^(#.*|.*@.*|.*https?|//.*|\d+([a-zA-Z]{2})?)$')
+my_punct = ''.join(x for x in string.punctuation if x not in ['#', '@'])
+trans = str.maketrans(my_punct, ' ' * len(my_punct))
```

Comment:
`maketrans` is pretty cool

Reply:
It's something I knew about but haven't used before… I know that there is a lot more flexibility to it and will be having a play.